src/SmartComponents/CompletedTaskDetails/JobResultsDetails/EntryRow.js

```diff
@@ -1,19 +1,23 @@
 import React from 'react';
 import propTypes from 'prop-types';
 import severityMap from '../TaskEntries';
-import { Icon } from '@patternfly/react-core';
+import EntryRowLabel from './EntryRowLabel';
 
 const EntryRow = ({ severity, taskConstantMapper, title }) => {
   const mappedSeverity =
     severityMap[taskConstantMapper][severity.toLowerCase()];
-  const renderIcon = () => {
+
+  const getLabelType = () => {
```

Comment:
Probably, this should be converted to a React component and be given proper component name.

Reply:
Done!